posts/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: password
+  web:
+    build: .
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails db:create && bundle exec rails db:migrate && bundle exec rails s -p 3000 -b '0.0.0.0'"
```

Comment:
A BD deve ser criada todas as vezes que for iniciado o projeto?

Reply:
Esta era uma solução temporária que deveria ter sido ajustada.
O ideal seria uma lógica (provavelmente em `entrypoint.sh`) para validar a criação de BD e não repetir o comando (mesmo ele sendo idempotente)